extensions/wikia/Email/Email.i18n.php

```diff
@@ -9,14 +9,6 @@
 	'emailext-anonymous-editor' => 'A Wikia fan',
 	'emailext-no-reply-name' => 'No Reply',
 	'emailext-fans-tagline' => 'The Social Universe for Fans by Fans',
```

Comment:
Just ripped these out since they aren't the right split of the messages and are the right copy and are in the wrong i18n file.